api/server.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
```

Comment:
I would like to challenge you with making a separate middleware file for your middleware rather than just putting it in your server.js